.github/workflows/release.yaml

```diff
@@ -0,0 +1,20 @@
+name: Release
+
+on:
+  release:
+    types:
+      - created
```

Comment:
I have couple of questions:
* How this works with draft releases? 
* What happens if we change release tag?

Reply:
> How this works with draft releases?

The workflow will not run for draft releases. That said, I switched from `created` to `published` since `created` was only triggering workflow for NON-draft releases. With `published` we can now have `release` and `pre-release` from draft release that will trigger the workflow.

> What happens if we change release tag?

This can only happen on releases that are still draft and unpublished, so this shouldn't be a problem.

Reply:
SG